src/couch/src/couch_debug.erl

```diff
@@ -615,6 +632,16 @@ restart_busy(ProcessList, Threshold, DelayInMsec, Property) when
 %%  - left
 %%  - centre
 %%  - right
+print_table(Rows, TableSpec) ->
+    io:format("~s~n", [format(TableSpec)]),
+    lists:foreach(
+        fun({Id, Props}) ->
+            io:format("~s~n", [table_row(Id, 2, Props, TableSpec)])
+        end,
+        Rows
+    ),
+    ok.
+
 print_tree(Tree, TableSpec) ->
```

Comment:
both print_table and print_tree should have comments documenting their arguments.

Reply:
Added.